frontend/src/scenes/notebooks/Notebook/notebooksListLogic.ts

```diff
@@ -44,16 +48,33 @@ export const openNotebook = (
     }
 
     popoverLogic?.actions.setInitialAutofocus(focus)
+
+    const theMountedNotebookLogic = notebookLogic({ shortId: notebookId })
```

Comment:
This doesn't actually mount it though right? Do we not ned to do `notebookLogic.findMounted()` ?